test/src/tests-forwarders.js

```diff
@@ -1534,9 +1654,9 @@ describe('forwarders', function() {
 
         done();
     });
+    });
 
     it('should pass in app version to forwarder on initialize', function(done) {
-        mParticle._resetForTests(MPConfig);
```

Comment:
I see you're removing this reset. any particular reason for this?

Reply:
same as above